docs/talmosang/talmosang-design/web-brief.md

```
@@ -0,0 +1,1393 @@
+**근본 원인**:
+```json
+// package.json
+"tailwindcss": "^4.0.0"  // v4 설치됨
+```
+
+```css
+/* globals.css */
+@import "tailwindcss";   // v4 문법 사용 중
+```
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)